services/static-webserver/client/source/class/osparc/desktop/MainPage.js

```diff
@@ -190,8 +190,34 @@ qx.Class.define("osparc.desktop.MainPage", {
           }, this);
         }
       });
+      studyBrowser.addListener("publishTemplate", e => this.__publishTemplate(e.getData()));
+    },
+
+    __publishTemplate: function(data) {
+      const text = this.tr("Templating process started and added to the background tasks");
```

Comment:
I'd change "Templating process started and added to the background tasks"" with "Started template creation and added to the background tasks"

Reply:
Done 👌